Lib/http/server.py

```diff
@@ -1251,6 +1252,46 @@ def run_cgi(self):
                 self.log_message("CGI script exited OK")
 
 
+class HTTPSServer(HTTPServer):
```

Comment:
Can you put it close to `HTTPServer` and not at the end of the file please?

Reply:
moved :+1: